.github/workflows/monitor_api_integration_test.yml

```diff
@@ -1,8 +1,6 @@
 name: monitor-api-integration-test
 
 on:
-  push:
-  pull_request:
```

Comment:
Another workflow `integration_tests.yml` already runs the monitor API test on `push` and `pull_request`, so no need to do it here.